test/DGmethods/integral_test_sphere.jl

```diff
@@ -130,7 +129,7 @@ let
   expected_result = [6.228615762850257e-7
                      9.671308320438864e-9
                      1.5102832678375277e-10
-                     2.359860999112363e-12]
+                     2.3598612210570662e-12]
```

Comment:
Slurm doesn't like this change: 
https://gist.github.com/climabot/f8b1957d388a35030d5bcc272c22e0dd#file-out_7303963-L43

Reply:
@jkozdon any reason for this change?

Reply:
```suggestion
                     2.359860999112363e-12]
```

Reply:
Ah, so the issue is that the CPU and GPU give different answers. Is this simply an order-of-operations issue, or something more serious. Can we just loosen the tolerance?

Reply:
Parallel vs serial reduction perhaps? If so, typical solution is to loosen the tolerance.